Dockerfile

```diff
@@ -1,5 +1,7 @@
 # Build the manager binary
 FROM golang:1.20 as builder
+ARG TARGETOS
+ARG TARGETARCH
```

Comment:
The default kubebuilder project now has multi-arch Docker image support.

Related to:
 * https://github.com/cert-manager/sample-external-issuer/pull/29

But I think I will have to modify the release workflow to use the new `make docker-buildx` target....in a separate PR.